RELEASE/scripts/sl_ascend.ash

```diff
@@ -12580,6 +12585,13 @@ boolean L9_oilPeak()
 	{
 		buffMaintain($effect[Ceaseless Snarling], 0, 1, 1);
 	}
+	// Maximize Asdon usage
+	if(((monster_level_adjustment() >= 75) && (monster_level_adjustment() <= 99)) || ((monster_level_adjustment() >= 25) && (monster_level_adjustment() <= 49)) || (monster_level_adjustment() <= 11))
+	{
+		asdonBuff($effect[Driving Recklessly]);
+	} else {
```

Comment:
Minor formatting nitpick, expand this to three lines to be consistent with most of the rest of the codebase please.